test/integrations/destinations/google_adwords_offline_conversions/dataDelivery/data.ts

```diff
@@ -1,750 +1,1064 @@
-                                                {
-                                                    "hashedEmail": "6db61e6dcbcf2390e4a46af426f26a133a3bee45021422fc7ae86e9136f14110",
-                                                    "userIdentifierSource": "UNSPECIFIED"
-                                                }
-                                            ]
-                                        }
-                                    }
-                                ],
-                                "validate_only": false
-                            },
-                            "createJobPayload": {
-                                "job": {
-                                    "storeSalesMetadata": {
-                                        "custom_key": "CUSTOM_KEY",
-                                        "loyaltyFraction": 1,
-                                        "transaction_upload_fraction": "1"
-                                    },
-                                    "type": "STORE_SALES_UPLOAD_FIRST_PARTY"
-                                }
-                            },
-                            "event": "1112223333",
-                            "executeJobPayload": {
-                                "validate_only": false
-                            },
-                            "isStoreConversion": true
-                        },
-                        "JSON_ARRAY": {},
-                        "XML": {},
-                        "FORM": {}
+    output: {
+      response: {
+        status: 401,
+        body: {
+          output: {
+            status: 401,
+            message:
+              '[Google Ads Offline Conversions]:: Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project. during google_ads_offline_store_conversions Job Creation',
+            authErrorCategory: 'REFRESH_TOKEN',
+            destinationResponse: {
+              error: {
+                code: 401,
+                message:
+                  'Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.',
+                status: 'UNAUTHENTICATED',
+              },
+            },
+            statTags: {
+              destType: 'GOOGLE_ADWORDS_OFFLINE_CONVERSIONS',
+              destinationId: 'Non-determininable',
+              errorCategory: 'network',
+              errorType: 'aborted',
+              feature: 'dataDelivery',
+              implementation: 'native',
+              module: 'destination',
+              workspaceId: 'Non-determininable',
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    name: 'google_adwords_offline_conversions',
+    description: 'Test 3',
```

Comment:
Can you please add description about what this test is about?

Reply:
it was old test, will update